### ISSUE TITLE
Package import does not work in Unity

### ISSUE THREAD

**Issue by mcdeimon** [Nethereum/Unity3dSimpleSampleNet461#4]
Hi! I am trying to import the package into Unity using the option "Add package from Github URL ...
"but it doesn't recognize the .json file.

Thanks!

**Comment by juanfranblanco** [comment]
No, there is no package as such for unity. I need to look into it. @mcdeimon
